e2e/test/scenarios/dashboard/dashboard-card-resizing.cy.spec.js

```diff
@@ -0,0 +1,183 @@
+];
+
+describe("scenarios > dashboard card resizing", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should display all visualization cards with their default sizes", () => {
+    TEST_QUESTIONS.forEach(question => {
+      cy.createQuestion(question);
+    });
+    cy.createDashboard().then(({ body: { id: dashId } }) => {
+      visitDashboard(dashId);
+
+      cy.findByTestId("dashboard-header").within(() => {
+        cy.findByLabelText("Edit dashboard").click();
+        cy.findByLabelText("Add questions").click();
+      });
+
+      TEST_QUESTIONS.forEach(question => {
+        cy.findByLabelText(question.name).click();
+      });
+
+      saveDashboard();
+
+      cy.request("GET", `/api/dashboard/${dashId}`).then(({ body }) => {
+        body.ordered_cards.forEach(({ card, size_x, size_y }) => {
+          const { width, height } = getDefaultSize(card.display);
+          expect(size_x).to.equal(width);
+          expect(size_y).to.equal(height);
+        });
+      });
+    });
+  });
+
+  it(`should not allow cards to be resized smaller than min height`, () => {
+    const cardIds = [];
+    TEST_QUESTIONS.forEach(question => {
+      cy.createQuestion(question).then(({ body: { id } }) => {
+        cardIds.push(id);
+      });
+    });
+    cy.createDashboard().then(({ body: { id: dashId } }) => {
+      cy.request("PUT", `/api/dashboard/${dashId}/cards`, {
+        cards: cardIds.map((cardId, index) => ({
+          id: index,
+          card_id: cardId,
+          row: index * 2,
+          col: 0,
+          size_x: 2,
+          size_y: 2,
+        })),
+      });
+      visitDashboard(dashId);
+      editDashboard();
+
+      cy.request("GET", `/api/dashboard/${dashId}`).then(({ body }) => {
+        const orderedCards = body.ordered_cards;
+        orderedCards.forEach(({ card }) => {
+          const dashCard = cy.contains(".DashCard", card.name);
+          dashCard.within(() => {
+            const resizeHandle = cy.get(".react-resizable-handle");
+            resizeHandle
+              .trigger("mousedown", { button: 0 })
+              .trigger("mousemove", {
+                clientX: getDefaultSize(card.display).width * 100,
+                clientY: getDefaultSize(card.display).height * 100,
+              })
+              .trigger("mouseup", { force: true });
+          });
+        });
+
+        saveDashboard();
+        editDashboard();
+
+        orderedCards.forEach(({ card }) => {
+          const dashCard = cy.contains(".DashCard", card.name);
+          dashCard.within(() => {
+            const resizeHandle = cy.get(".react-resizable-handle");
+            resizeHandle
+              .trigger("mousedown", { button: 0 })
+              .trigger("mousemove", {
+                clientX: -getDefaultSize(card.display).width * 100,
+                clientY: -getDefaultSize(card.display).height * 100,
+              })
+              .trigger("mouseup", { force: true });
```

Comment:
this would be a decent candidate for a little helper function like `resizeDashcard({ resizeHandle, x, y});`